### DISCUSSION TITLE
Uploading an analysis to your portfolio demo video

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/Earth-Analytics-2023-01-Intro#214] [Technical Demo Videos]
Here it is! @earthlab-education/ea-students @earthlab-education/ea-instructors 

https://cuboulder.zoom.us/rec/share/SG7RTxqDJuNfA5NzkvbmqmSXXE1AezrnS6R3KfW7M3atoXBdJy9cw4bsa-jj6fqQ.QNYFMmssw-4CCApC

You don't have to follow this method, but I like it because it will automatically update your webpage should you change and re-run your code. I'm not concerned about the solutions being public, but you can make your portfolio repository private if you wish without taking your website down.

**Comment by edchanonline** [comment]
Tysm! Love this - implemented my own flavor at https://eggvoice.github.io/#san-francisco-average-temperatures
